website/versioned_docs/version-2.0.0-alpha.70/markdown-features.mdx

```diff
@@ -1072,7 +1072,7 @@ The ES imports syntax also works:
 
 import myImageUrl from './assets/docusaurus-asset-example-banner.png';
 
-<img src={myImageUrl)} alt="My image alternative text" />
+<img src={myImageUrl} alt="My image alternative text" />
```

Comment:
thanks

Can you do that change in /website/docs/markdown-features.mdx too please?

Reply:
Good catch 👍

Reply:
Did you mean by `website/docs/guides/markdown-features/markdown-features-assets.mdx` right @slorber ?

Reply:
yes, the same file in the upstream version so that it's not lost on alpha 71 when we release it ;)